### ISSUE TITLE
High CPU usage on Ubuntu 22.04 WSL

### ISSUE THREAD

**Issue by Wyzix33** [cirospaciari/socketify.py#112]
**Describe the bug**
On a fresh Ubuntu 22.04 installed as a Windows Subsystem for Linux (WSL) i'm seeing 
```
PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND
814 root      20   0   34516  24624  10804 R 100.0   0.3   0:15.78 python3
```
for as long as the server is running

**To Reproduce**
create a server.py file containing
```
from socketify import App
app = App()
router = app.router()
app.listen(
    8080,
    lambda config: print(
        "Listening on port http://localhost:%d now\n" % config.port),
)
app.run()
```

**Expected behavior**
Normal CPU usage

**Screenshots**
![Screenshot (19)](https://user-images.githubusercontent.com/13553412/224483572-f8429659-3a15-4535-83f9-001cc91df3c0.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04 installed as WSL

**Additional context**
Python 3.10.6
[socketify.py](https://github.com/cirospaciari/socketify.py)  v0.0.17

PS on a normal installation of Ubuntu 22.04 it's OK

**Comment by cirospaciari** [comment]
Never tried on WSL before. I will setup a dual boot here and try it, Thanks for reporting, this type of feedback is essential!

**Comment by Wyzix33** [comment]
WSL on Windows 10 22H2, forgot to mention

**Comment by eirnym** [comment]
@Wyzix33 WSL or WSL2?

**Comment by Wyzix33** [comment]
```
PS C:\Users\G> wsl -l -v
  NAME            STATE           VERSION
* Ubuntu-22.04    Running         2
```

**Comment by Wyzix33** [comment]
Also there is the same problem on colab 
![Screenshot (21)](https://user-images.githubusercontent.com/13553412/224491545-8756aa84-f0a5-4352-ac41-3da54163c242.png)

**Comment by cirospaciari** [comment]
fixed on v0.0.19
